MediaBrowser.Model/Updates/PackageInfo.cs

```diff
@@ -32,24 +26,6 @@ public class PackageInfo
         /// <value>The overview.</value>
         public string overview { get; set; }
 
-        /// <summary>
-        /// Gets or sets a value indicating whether this instance is premium.
-        /// </summary>
-        /// <value><c>true</c> if this instance is premium; otherwise, <c>false</c>.</value>
-        public bool isPremium { get; set; }
-
-        /// <summary>
-        /// Gets or sets a value indicating whether this instance is adult only content.
-        /// </summary>
-        /// <value><c>true</c> if this instance is adult; otherwise, <c>false</c>.</value>
-        public bool adult { get; set; }
-
-        /// <summary>
-        /// Gets or sets the rich desc URL.
-        /// </summary>
-        /// <value>The rich desc URL.</value>
-        public string richDescUrl { get; set; }
```

Comment:
why remove?

Reply:
We already have too many description properties and nothing uses the rich description. This field would also be useless for most clients, and I'd rather not implement it in the web client, so it will probably stay unused.